content/docs/command-reference/metrics/diff.md

```diff
@@ -43,7 +43,21 @@ lists all the current metrics without comparisons.
 ## Options
 
 - `--targets <paths>` - limit command scope to these metric files. Using -R,
-  directories to search metric files in can also be given.
+  directories to search metric files in can also be given. When specifying
+  arguments for `--targets` before `revisions`, you may use `--` after this
+  option's arguments, e.g.:
+
+  ```dvc
+  $ dvc metrics diff --targets t1.json t2.yaml -- HEAD v1
+  ```
+
+  Without `--`, this statement will show an error as argument parser confuses
+  `revisions` as arguments for `--targets` option. Alternatively, you can also
+  run above statement as:
```

Comment:
Let's just avoid this sentence and change "may" for "must" or "should" in the previous one, please. Also, "the" is missing again 😞 

```suggestion
  Alternatively, you can also run the above statement as:
```

Same for plots diff, tnx